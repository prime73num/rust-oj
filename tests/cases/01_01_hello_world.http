###
# Send request:
POST http://127.0.0.1:12345/jobs HTTP/1.1
content-type: application/json

{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World!\"); }","user_id":0}
###
# Got response:
# HTTP 200
# content-length: 417
# content-type: application/json
# date: Sat, 03 Sep 2022 04:02:02 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Compilation Success","time":0},{"id":1,"info":"","memory":0,"result":"Accepted","time":0}],"created_time":"2022-09-03T04:02:02.522Z","id":0,"result":"Accepted","score":100.0,"state":"Finished","submission":{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World!\"); }","user_id":0},"updated_time":"2022-09-03T04:02:02.522Z"}
