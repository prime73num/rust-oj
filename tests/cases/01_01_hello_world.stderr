[2022-09-03T05:07:51Z INFO  actix_server::builder] Starting 2 workers
[2022-09-03T05:07:51Z INFO  actix_server::server] Actix runtime found; starting in Actix runtime
[2022-09-03T05:07:52Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET / HTTP/1.1" 404 0 "-" "-" 0.000298
[2022-09-03T05:07:52Z INFO  post_jobs_handler] Post jobs
[2022-09-03T05:07:53Z INFO  post_jobs_handler] job run success
[2022-09-03T05:07:53Z INFO  actix_web::middleware::logger] 127.0.0.1 "POST /jobs HTTP/1.1" 200 417 "-" "-" 0.310356
